ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -115,24 +120,32 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
         );
     }
 
-    private fetchWorkflows(): void {
+    private fetchWorkflows(namespace: string, selectedPhases: string[], selectedLabels: string[]): void {
```

Comment:
Change to use passed in variables because it seems like `setState` is async and using `this.state.xxx` right after `setState` does not return latest value.